apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -0,0 +1,288 @@
+import { Injectable } from '@nestjs/common'
+import {
+  ConnectProfileStatus,
+  MentorshipMatchStatus,
+  RediLocation,
+  UserType,
+} from '@talent-connect/common-types'
+import * as AWS from 'aws-sdk'
+import * as jsforce from 'jsforce'
+import { ConMentorshipMatchesService } from '../con-mentorship-matches/con-mentorship-matches.service'
+import { ConProfilesService } from '../con-profiles/con-profiles.service'
+import { SfApiEmailTemplatesService } from '../salesforce-api/sf-api-email-templates.service'
+
+const sesConfig = {
+  accessKeyId: process.env.NX_EMAILER_AWS_ACCESS_KEY,
+  secretAccessKey: process.env.NX_EMAILER_AWS_SECRET_KEY,
+  region: process.env.NX_EMAILER_AWS_REGION,
+}
+
+const isProductionOrDemonstration = () =>
+  ['production', 'demonstration', 'staging'].includes(process.env.NODE_ENV)
+
+@Injectable()
+export class ReminderEmailsService {
+  private ses: AWS.SES
+
+  constructor(
+    private readonly emailTemplatesService: SfApiEmailTemplatesService,
+    private readonly conProfilesService: ConProfilesService,
+    private readonly conMentorshipMatchesService: ConMentorshipMatchesService
+  ) {
+    this.ses = new AWS.SES(sesConfig)
+  }
+
+  async getDraftingConProfiles({ userType }: { userType: UserType }) {
+    if (!userType) {
+      throw new Error('User type is required')
+    }
+
+    const createdDate = new Date()
+    createdDate.setDate(createdDate.getDate() - 14)
+    createdDate.setHours(0, 0, 0, 0) // set time to 00:00
+
+    const endOfCreatedDate = new Date(createdDate)
+    endOfCreatedDate.setHours(23, 59, 59, 999) // set time to 23:59
+
+    return this.conProfilesService.findAll({
+      'RecordType.DeveloperName': userType,
+      Profile_Status__c: ConnectProfileStatus.DRAFTING_PROFILE,
+      CreatedDate: {
```

Comment:
I created a new formula field (in all SF Orgs) called `Created_Date_Without_Time__c	`. You can use that to simplify this code using the simpler logic like the one down in `getApprovedMenteesByDaysAndLocation()`

Reply:
I don't know why you hated this, this much hahah. But thanks then I'll switch to that 💪